server/chat-commands/admin.ts

```diff
@@ -420,14 +420,17 @@ export const commands: ChatCommands = {
 				void IPTools.loadHostsAndRanges();
 				this.sendReply("IPTools has been hot-patched.");
 			} else if (target === 'modlog') {
+				if (!Config.usesqlite) {
```

Comment:
I'd just kill this block of code for now. Until we actually move everything over to SQLite the existing text modlogs should continue to work without interruption.